documentation/latest/operations/remote_data_store.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: documentation
+---
+
+# Remote data store for Flotta Edge API
+
+## Introduction
+
+By default Flotta Edge API uses Kubernetes and CRD-based storage as a source of truth about managed Edge entities -
+`EdgeDevice` and `EdgeWorkload`. All the other CRDs serve as support for better device management in Kubernetes
+environment.
+
+![](../diagrams/edge_api.png)
+
+Users might want to store their device configuration data in some other place than Kubernetes etcd. To support that
+use-case we provide a way to use that external data store (or to be precise - data provider service) in Flotta Edge API.
+
+On the diagram below you can see *Flotta Edge API* still providing data to a *Device* in the same way as it is happening
+now. The difference is that Device configuration (including workload definitions) is read from an external service
+represented by *Edge Configuration API*, which in turn uses its own data store - *EdgeConfigurationProvider*. It is
+responsible for providing said Device configuration in format defined by Flotta. The heartbeat messages will be sent to
+*Edge Configuration API* as well. *etcd* on the diagram is used only for storing objects being under supervision of
+*Flotta Edge API*, like Secrets with mTLS certificates used for *Device-Flotta Edge API* communication.
+
+**Warning!**
+
+*Flotta Operator can't use custom, external data store and its functionality, if needed, has to be implemented by the
+users*
+
+![](../diagrams/edge_api_remote_db.png)
+
+## Flotta Edge API configuration for remote data store
+
+### Prerequisites
+
+* HTTP service
+  providing [Edge Configuration API](../operations/configuration-api.html).
+  You can take inspiration from
+  our [integration test](https://github.com/project-flotta/flotta-operator/blob/main/internal/edgeapi/backend/remote/backend_integration_test.go)
+  and
+  a [fake server implementation](https://github.com/project-flotta/flotta-operator/blob/main/internal/edgeapi/backend/remote/fake-server_test.go)
+  ;
+* Flotta Edge API deployed - as part of the whole Flotta deployment or a standalone deployment (Flotta operator would be
+  useless);
+* Edge Configuration API server has to be reachable from Flotta Edge API pod;
+
+### Flotta Edge API configuration
+
+Flotta Edge API configuration can be found in `flotta-edge-api-config` ConfigMap in `flotta` namespace. To configure
```

Comment:
is there a planned PR for that? I haven't found a reference to this configmap in our operator's repository or in code.

Reply:
It's there: https://github.com/project-flotta/flotta-operator/blob/3ea8160dcadab2a556c9e179bd5ddc8bc3a3d458/config/edge-api/kustomization.yaml#L7

Reply:
right. was looking for the name post-processing by kustomize :-)